CMakeLists.txt

```
@@ -188,12 +188,17 @@ if(KOKKOSTOOLS_HAS_VARIORUM)
 endif()
 
 # GPU profilers
-if(Kokkos_ENABLE_CUDA)
+if(KOKKOSTOOLS_HAS_NVTX)
   add_subdirectory(profiling/nvtx-connector)
   add_subdirectory(profiling/nvtx-focused-connector)
+else()
+  message(STATUS "Skipping nvtx-connector:         NVTX is not enabled (maybe Kokkos was not found, or not configured with the CUDA backend)")
```

I feel like that the message is potentially confusing, I would do just
```suggestion
  message(STATUS "Skipping nvtx-connector (NVTX disabled)")
```